tests/src/python/test_qgsannotationlayer.py

```diff
@@ -58,13 +58,15 @@ class TestQgsAnnotationLayer(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
+        super().setUpClass()
         cls.report = "<h1>Python QgsAnnotationLayer Tests</h1>\n"
 
     @classmethod
     def tearDownClass(cls):
         report_file_path = "%s/qgistest.html" % QDir.tempPath()
```

Comment:
Is this still usefull ? the report should be wrote by write_local_html_report. same comment for all other test methods where *qgistest.html* is generated